llvm/test/MachineVerifier/test_abd_su.mir

```diff
@@ -6,6 +6,14 @@ name:            g_abd_su
 body: |
   bb.0:
 
+    %2:_(p0) = G_IMPLICIT_DEF
+    %3:_(p0) = G_IMPLICIT_DEF
+    %4:_(s1) = G_ABDS %2, %3
```

Comment:
Missing the error check?

Reply:
I removed the C++ error checking code under the assumption that the parameters are types and not ptypes and some other part of the system checks for misbehaviour.

Reply:
But that does not remove the need to check that the correct error is produced. I also see the case with a pointer type in each operand is not rejected